teamcity: failed tests on v2.0-alpha.20180116: test/TestRestoreDatabaseVersusTable, test/TestRestoreDatabaseVersusTable/into_db, test/TestRestoreDatabaseVersusTable/incomplete-db, test/TestRestoreDatabaseVersusTable/tables, test/TestRestoreDatabaseVersusTable/db-exists, test/TestRestoreDatabaseVersusTable/db, test/TestRestoreDatabaseVersusTable/tables-needs-db

The following tests appear to have failed:

[#480343](https://teamcity.cockroachdb.com/viewLog.html?buildId=480343):

```
--- FAIL: test/TestRestoreDatabaseVersusTable (0.000s)
Test ended in panic.

------- Stdout: -------
W180116 16:28:44.374977 148702 server/status/runtime.go:109  Could not parse build timestamp: parsing time "" as "2006/01/02 15:04:05": cannot parse "" as "2006"
I180116 16:28:44.377172 148702 server/config.go:516  [n?] 1 storage engine initialized
I180116 16:28:44.377195 148702 server/config.go:519  [n?] RocksDB cache size: 128 MiB
I180116 16:28:44.377201 148702 server/config.go:519  [n?] store 0: in-memory, size 0 B
I180116 16:28:44.380489 148702 server/node.go:361  [n?] **** cluster 2ffe2780-b05d-49fe-a57a-d95539f9cf40 has been created
I180116 16:28:44.380533 148702 server/server.go:934  [n?] **** add additional nodes by specifying --join=127.0.0.1:45707
I180116 16:28:44.381352 148702 storage/store.go:1209  [n1,s1] [n1,s1]: failed initial metrics computation: [n1,s1]: system config not yet available
I180116 16:28:44.381408 148702 server/node.go:486  [n1] initialized store [n1,s1]: disk (capacity=512 MiB, available=512 MiB, used=0 B, logicalBytes=3.2 KiB), ranges=1, leases=0, writes=0.00, bytesPerReplica={p10=3322.00 p25=3322.00 p50=3322.00 p75=3322.00 p90=3322.00}, writesPerReplica={p10=0.00 p25=0.00 p50=0.00 p75=0.00 p90=0.00}
I180116 16:28:44.381440 148702 server/node.go:339  [n1] node ID 1 initialized
I180116 16:28:44.381493 148702 gossip/gossip.go:332  [n1] NodeDescriptor set to node_id:1 address:<network_field:"tcp" address_field:"127.0.0.1:45707" > attrs:<> locality:<> ServerVersion:<major_val:1 minor_val:1 patch:0 unstable:8 > 
I180116 16:28:44.381664 148702 storage/stores.go:331  [n1] read 0 node addresses from persistent storage
I180116 16:28:44.381776 148702 server/node.go:627  [n1] connecting to gossip network to verify cluster ID...
I180116 16:28:44.381802 148702 server/node.go:652  [n1] node connected via gossip and verified as part of cluster "2ffe2780-b05d-49fe-a57a-d95539f9cf40"
I180116 16:28:44.381836 148702 server/node.go:428  [n1] node=1: started with [<no-attributes>=<in-mem>] engine(s) and attributes []
I180116 16:28:44.381891 148702 sql/distsql_physical_planner.go:122  [n1] creating DistSQLPlanner with address {tcp 127.0.0.1:45707}
I180116 16:28:44.383402 148967 storage/replica_command.go:819  [split,n1,s1,r1/1:/M{in-ax}] initiating a split of this range at key /System/"" [r2]
I180116 16:28:44.390026 148702 server/server.go:1161  [n1] starting https server at 127.0.0.1:39153
I180116 16:28:44.390056 148702 server/server.go:1162  [n1] starting grpc/postgres server at 127.0.0.1:45707
I180116 16:28:44.390066 148702 server/server.go:1163  [n1] advertising CockroachDB node at 127.0.0.1:45707
E180116 16:28:44.391004 148968 storage/queue.go:668  [replicate,n1,s1,r1/1:/{Min-System/}] range requires a replication change, but lacks a quorum of live replicas (0/1)
I180116 16:28:44.392057 148967 storage/replica_command.go:819  [split,n1,s1,r2/1:/{System/-Max}] initiating a split of this range at key /System/NodeLiveness [r3]
I180116 16:28:44.398783 148967 storage/replica_command.go:819  [split,n1,s1,r3/1:/{System/NodeL…-Max}] initiating a split of this range at key /System/NodeLivenessMax [r4]
W180116 16:28:44.399561 149047 storage/intent_resolver.go:324  [n1,s1] failed to push during intent resolution: failed to push "sql txn implicit" id=1f3102da key=/Table/SystemConfigSpan/Start rw=true pri=0.01228340 iso=SERIALIZABLE stat=PENDING epo=0 ts=1516120124.393517247,0 orig=1516120124.393517247,0 max=1516120124.393517247,0 wto=false rop=false seq=7
I180116 16:28:44.400055 148702 sql/event_log.go:115  [n1] Event: "alter_table", target: 12, info: {TableName:eventlog Statement:ALTER TABLE system.eventlog ALTER COLUMN "uniqueID" SET DEFAULT uuid_v4() User:node MutationID:0 CascadeDroppedViews:[]}
I180116 16:28:44.403524 148967 storage/replica_command.go:819  [split,n1,s1,r4/1:/{System/NodeL…-Max}] initiating a split of this range at key /System/tsd [r5]
I180116 16:28:44.404276 148702 sql/lease.go:348  [n1] publish: descID=12 (eventlog) version=2 mtime=2018-01-16 16:28:44.404137559 +0000 UTC
I180116 16:28:44.411017 148967 storage/replica_command.go:819  [split,n1,s1,r5/1:/{System/tsd-Max}] initiating a split of this range at key /System/"tse" [r6]
I180116 16:28:44.417412 148967 storage/replica_command.go:819  [split,n1,s1,r6/1:/{System/tse-Max}] initiating a split of this range at key /Table/SystemConfigSpan/Start [r7]
I180116 16:28:44.420306 148702 sql/event_log.go:115  [n1] Event: "set_cluster_setting", target: 0, info: {SettingName:diagnostics.reporting.enabled Value:true User:node}
I180116 16:28:44.421670 148967 storage/replica_command.go:819  [split,n1,s1,r7/1:/{Table/System…-Max}] initiating a split of this range at key /Table/11 [r8]
I180116 16:28:44.425632 148967 storage/replica_command.go:819  [split,n1,s1,r8/1:/{Table/11-Max}] initiating a split of this range at key /Table/12 [r9]
I180116 16:28:44.429319 148967 storage/replica_command.go:819  [split,n1,s1,r9/1:/{Table/12-Max}] initiating a split of this range at key /Table/13 [r10]
I180116 16:28:44.430672 148702 sql/event_log.go:115  [n1] Event: "set_cluster_setting", target: 0, info: {SettingName:version Value:$1 User:node}
I180116 16:28:44.433935 148967 storage/replica_command.go:819  [split,n1,s1,r10/1:/{Table/13-Max}] initiating a split of this range at key /Table/14 [r11]
I180116 16:28:44.435240 148702 sql/event_log.go:115  [n1] Event: "set_cluster_setting", target: 0, info: {SettingName:trace.debug.enable Value:false User:node}
I180116 16:28:44.440327 148967 storage/replica_command.go:819  [split,n1,s1,r11/1:/{Table/14-Max}] initiating a split of this range at key /Table/15 [r12]
I180116 16:28:44.444271 148967 storage/replica_command.go:819  [split,n1,s1,r12/1:/{Table/15-Max}] initiating a split of this range at key /Table/16 [r13]
I180116 16:28:44.447890 148967 storage/replica_command.go:819  [split,n1,s1,r13/1:/{Table/16-Max}] initiating a split of this range at key /Table/17 [r14]
I180116 16:28:44.449926 148702 sql/event_log.go:115  [n1] Event: "alter_table", target: 4, info: {TableName:users Statement:ALTER TABLE system.users ADD COLUMN IF NOT EXISTS "isRole" BOOL NOT NULL DEFAULT false User:node MutationID:1 CascadeDroppedViews:[]}
I180116 16:28:44.451621 148967 storage/replica_command.go:819  [split,n1,s1,r14/1:/{Table/17-Max}] initiating a split of this range at key /Table/18 [r15]
I180116 16:28:44.452066 148702 sql/lease.go:348  [n1] publish: descID=4 (users) version=2 mtime=2018-01-16 16:28:44.451968724 +0000 UTC
I180116 16:28:44.455304 148967 storage/replica_command.go:819  [split,n1,s1,r15/1:/{Table/18-Max}] initiating a split of this range at key /Table/19 [r16]
I180116 16:28:44.456410 148702 sql/lease.go:348  [n1] publish: descID=4 (users) version=3 mtime=2018-01-16 16:28:44.456312308 +0000 UTC
I180116 16:28:44.458521 148967 storage/replica_command.go:819  [split,n1,s1,r16/1:/{Table/19-Max}] initiating a split of this range at key /Table/20 [r17]
I180116 16:28:44.459681 148702 sql/backfill.go:132  [n1] Running backfill for "users", v=3, m=1
I180116 16:28:44.461195 148967 storage/replica_command.go:819  [split,n1,s1,r17/1:/{Table/20-Max}] initiating a split of this range at key /Table/21 [r18]
I180116 16:28:44.467606 148967 storage/replica_command.go:819  [split,n1,s1,r18/1:/{Table/21-Max}] initiating a split of this range at key /Table/22 [r19]
I180116 16:28:44.468667 148702 sql/lease.go:348  [n1] publish: descID=4 (users) version=4 mtime=2018-01-16 16:28:44.468575266 +0000 UTC
I180116 16:28:44.470853 148967 storage/replica_command.go:819  [split,n1,s1,r19/1:/{Table/22-Max}] initiating a split of this range at key /Table/23 [r20]
I180116 16:28:44.472250 148702 sql/event_log.go:115  [n1] Event: "finish_schema_change", target: 4, info: {MutationID:1}
I180116 16:28:44.473551 148702 sql/lease.go:274  publish (count leases): descID=4 name=users version=3 count=1
I180116 16:28:44.499654 148702 server/server.go:1232  [n1] done ensuring all necessary migrations have run
I180116 16:28:44.499690 148702 server/server.go:1235  [n1] serving sql connections
I180116 16:28:44.501981 149389 sql/event_log.go:115  [n1] Event: "node_join", target: 1, info: {Descriptor:{NodeID:1 Address:{NetworkField:tcp AddressField:127.0.0.1:45707} Attrs: Locality: ServerVersion:1.1-8} ClusterID:2ffe2780-b05d-49fe-a57a-d95539f9cf40 StartedAt:1516120124381809901 LastUp:1516120124381809901}
I180116 16:28:44.512797 149363 sql/event_log.go:115  [client=127.0.0.1:53982,user=root,n1] Event: "create_database", target: 50, info: {DatabaseName:data Statement:CREATE DATABASE data User:root}
I180116 16:28:44.513124 148967 storage/replica_command.go:819  [split,n1,s1,r20/1:/{Table/23-Max}] initiating a split of this range at key /Table/50 [r21]
I180116 16:28:44.516594 149363 sql/event_log.go:115  [client=127.0.0.1:53982,user=root,n1] Event: "create_table", target: 51, info: {TableName:bank Statement:CREATE TABLE bank (id INT PRIMARY KEY, balance INT, payload STRING, FAMILY (id, balance, payload)) User:root}
I180116 16:28:44.516963 148967 storage/replica_command.go:819  [split,n1,s1,r21/1:/{Table/50-Max}] initiating a split of this range at key /Table/51 [r22]
I180116 16:28:44.523820 149363 sql/event_log.go:115  [client=127.0.0.1:53982,user=root,n1] Event: "create_database", target: 52, info: {DatabaseName:d2 Statement:CREATE DATABASE d2 User:root}
I180116 16:28:44.524163 148967 storage/replica_command.go:819  [split,n1,s1,r22/1:/{Table/51-Max}] initiating a split of this range at key /Table/52 [r23]
I180116 16:28:44.526841 149363 sql/event_log.go:115  [client=127.0.0.1:53982,user=root,n1] Event: "create_database", target: 53, info: {DatabaseName:d3 Statement:CREATE DATABASE d3 User:root}
I180116 16:28:44.527814 148967 storage/replica_command.go:819  [split,n1,s1,r23/1:/{Table/52-Max}] initiating a split of this range at key /Table/53 [r24]
I180116 16:28:44.530589 149363 sql/event_log.go:115  [client=127.0.0.1:53982,user=root,n1] Event: "create_table", target: 54, info: {TableName:d3.foo Statement:CREATE TABLE d3.foo (a INT) User:root}
I180116 16:28:44.531535 148967 storage/replica_command.go:819  [split,n1,s1,r24/1:/{Table/53-Max}] initiating a split of this range at key /Table/54 [r25]
I180116 16:28:44.533922 149363 sql/event_log.go:115  [client=127.0.0.1:53982,user=root,n1] Event: "create_database", target: 55, info: {DatabaseName:d4 Statement:CREATE DATABASE d4 User:root}
I180116 16:28:44.535456 148967 storage/replica_command.go:819  [split,n1,s1,r25/1:/{Table/54-Max}] initiating a split of this range at key /Table/55 [r26]
I180116 16:28:44.538408 149363 sql/event_log.go:115  [client=127.0.0.1:53982,user=root,n1] Event: "create_table", target: 56, info: {TableName:d4.foo Statement:CREATE TABLE d4.foo (a INT) User:root}
I180116 16:28:44.539645 148967 storage/replica_command.go:819  [split,n1,s1,r26/1:/{Table/55-Max}] initiating a split of this range at key /Table/56 [r27]
I180116 16:28:44.542084 149363 sql/event_log.go:115  [client=127.0.0.1:53982,user=root,n1] Event: "create_table", target: 57, info: {TableName:d4.bar Statement:CREATE TABLE d4.bar (a INT) User:root}
I180116 16:28:44.543349 148967 storage/replica_command.go:819  [split,n1,s1,r27/1:/{Table/56-Max}] initiating a split of this range at key /Table/57 [r28]
I180116 16:28:44.552426 149493 ccl/storageccl/export.go:124  [n1,s1,r27/1:/Table/5{6-7}] export [/Table/56/1,/Table/56/2)
I180116 16:28:44.552565 149492 ccl/storageccl/export.go:124  [n1,s1,r25/1:/Table/5{4-5}] export [/Table/54/1,/Table/54/2)
I180116 16:28:44.552588 149491 ccl/storageccl/export.go:124  [n1,s1,r22/1:/Table/5{1-2}] export [/Table/51/1,/Table/51/2)
I180116 16:28:44.552428 149494 ccl/storageccl/export.go:124  [n1,s1,r28/1:/{Table/57-Max}] export [/Table/57/1,/Table/57/2)
I180116 16:28:44.578698 149513 ccl/storageccl/export.go:124  [n1,s1,r27/1:/Table/5{6-7}] export [/Table/56/1,/Table/56/2)
I180116 16:28:44.594770 149524 ccl/storageccl/export.go:124  [n1,s1,r28/1:/{Table/57-Max}] export [/Table/57/1,/Table/57/2)
I180116 16:28:44.594837 149523 ccl/storageccl/export.go:124  [n1,s1,r27/1:/Table/5{6-7}] export [/Table/56/1,/Table/56/2)
I180116 16:28:44.613115 149501 ccl/storageccl/export.go:124  [n1,s1,r28/1:/{Table/57-Max}] export [/Table/57/1,/Table/57/2)
I180116 16:28:44.613208 149500 ccl/storageccl/export.go:124  [n1,s1,r27/1:/Table/5{6-7}] export [/Table/56/1,/Table/56/2)
--- FAIL: test/TestRestoreDatabaseVersusTable/into_db (0.000s)
Test ended in panic.

------- Stdout: -------
W180116 16:28:48.138581 157820 server/status/runtime.go:109  Could not parse build timestamp: parsing time "" as "2006/01/02 15:04:05": cannot parse "" as "2006"
I180116 16:28:48.145014 157820 server/config.go:516  [n?] 1 storage engine initialized
I180116 16:28:48.145042 157820 server/config.go:519  [n?] RocksDB cache size: 128 MiB
I180116 16:28:48.145048 157820 server/config.go:519  [n?] store 0: in-memory, size 0 B
I180116 16:28:48.148107 157820 server/node.go:361  [n?] **** cluster f8371979-8bbb-42c2-9861-24b11bebe89a has been created
I180116 16:28:48.148140 157820 server/server.go:934  [n?] **** add additional nodes by specifying --join=127.0.0.1:44949
I180116 16:28:48.149103 157820 storage/store.go:1209  [n1,s1] [n1,s1]: failed initial metrics computation: [n1,s1]: system config not yet available
I180116 16:28:48.149154 157820 server/node.go:486  [n1] initialized store [n1,s1]: disk (capacity=512 MiB, available=512 MiB, used=0 B, logicalBytes=3.2 KiB), ranges=1, leases=0, writes=0.00, bytesPerReplica={p10=3322.00 p25=3322.00 p50=3322.00 p75=3322.00 p90=3322.00}, writesPerReplica={p10=0.00 p25=0.00 p50=0.00 p75=0.00 p90=0.00}
I180116 16:28:48.149179 157820 server/node.go:339  [n1] node ID 1 initialized
I180116 16:28:48.149239 157820 gossip/gossip.go:332  [n1] NodeDescriptor set to node_id:1 address:<network_field:"tcp" address_field:"127.0.0.1:44949" > attrs:<> locality:<> ServerVersion:<major_val:1 minor_val:1 patch:0 unstable:8 > 
I180116 16:28:48.149337 157820 storage/stores.go:331  [n1] read 0 node addresses from persistent storage
I180116 16:28:48.149445 157820 server/node.go:627  [n1] connecting to gossip network to verify cluster ID...
I180116 16:28:48.149476 157820 server/node.go:652  [n1] node connected via gossip and verified as part of cluster "f8371979-8bbb-42c2-9861-24b11bebe89a"
I180116 16:28:48.149502 157820 server/node.go:428  [n1] node=1: started with [<no-attributes>=<in-mem>] engine(s) and attributes []
I180116 16:28:48.149561 157820 sql/distsql_physical_planner.go:122  [n1] creating DistSQLPlanner with address {tcp 127.0.0.1:44949}
I180116 16:28:48.151288 158031 storage/replica_command.go:819  [split,n1,s1,r1/1:/M{in-ax}] initiating a split of this range at key /System/"" [r2]
I180116 16:28:48.152525 157820 server/server.go:1161  [n1] starting https server at 127.0.0.1:39351
I180116 16:28:48.152551 157820 server/server.go:1162  [n1] starting grpc/postgres server at 127.0.0.1:44949
I180116 16:28:48.152561 157820 server/server.go:1163  [n1] advertising CockroachDB node at 127.0.0.1:44949
I180116 16:28:48.159122 158031 storage/replica_command.go:819  [split,n1,s1,r2/1:/{System/-Max}] initiating a split of this range at key /System/NodeLiveness [r3]
I180116 16:28:48.161934 157820 sql/event_log.go:115  [n1] Event: "alter_table", target: 12, info: {TableName:eventlog Statement:ALTER TABLE system.eventlog ALTER COLUMN "uniqueID" SET DEFAULT uuid_v4() User:node MutationID:0 CascadeDroppedViews:[]}
I180116 16:28:48.163240 158031 storage/replica_command.go:819  [split,n1,s1,r3/1:/{System/NodeL…-Max}] initiating a split of this range at key /System/NodeLivenessMax [r4]
I180116 16:28:48.166759 157820 sql/lease.go:348  [n1] publish: descID=12 (eventlog) version=2 mtime=2018-01-16 16:28:48.166589069 +0000 UTC
I180116 16:28:48.167948 158031 storage/replica_command.go:819  [split,n1,s1,r4/1:/{System/NodeL…-Max}] initiating a split of this range at key /System/tsd [r5]
I180116 16:28:48.172582 158031 storage/replica_command.go:819  [split,n1,s1,r5/1:/{System/tsd-Max}] initiating a split of this range at key /System/"tse" [r6]
I180116 16:28:48.181804 158031 storage/replica_command.go:819  [split,n1,s1,r6/1:/{System/tse-Max}] initiating a split of this range at key /Table/SystemConfigSpan/Start [r7]
I180116 16:28:48.183812 157820 sql/event_log.go:115  [n1] Event: "set_cluster_setting", target: 0, info: {SettingName:diagnostics.reporting.enabled Value:true User:node}
I180116 16:28:48.185801 158031 storage/replica_command.go:819  [split,n1,s1,r7/1:/{Table/System…-Max}] initiating a split of this range at key /Table/11 [r8]
I180116 16:28:48.190312 158031 storage/replica_command.go:819  [split,n1,s1,r8/1:/{Table/11-Max}] initiating a split of this range at key /Table/12 [r9]
I180116 16:28:48.194182 158031 storage/replica_command.go:819  [split,n1,s1,r9/1:/{Table/12-Max}] initiating a split of this range at key /Table/13 [r10]
I180116 16:28:48.195422 157820 sql/event_log.go:115  [n1] Event: "set_cluster_setting", target: 0, info: {SettingName:version Value:$1 User:node}
I180116 16:28:48.198615 158031 storage/replica_command.go:819  [split,n1,s1,r10/1:/{Table/13-Max}] initiating a split of this range at key /Table/14 [r11]
I180116 16:28:48.198916 157820 sql/event_log.go:115  [n1] Event: "set_cluster_setting", target: 0, info: {SettingName:trace.debug.enable Value:false User:node}
I180116 16:28:48.202975 158031 storage/replica_command.go:819  [split,n1,s1,r11/1:/{Table/14-Max}] initiating a split of this range at key /Table/15 [r12]
I180116 16:28:48.206413 158031 storage/replica_command.go:819  [split,n1,s1,r12/1:/{Table/15-Max}] initiating a split of this range at key /Table/16 [r13]
I180116 16:28:48.211016 158031 storage/replica_command.go:819  [split,n1,s1,r13/1:/{Table/16-Max}] initiating a split of this range at key /Table/17 [r14]
I180116 16:28:48.211425 157820 sql/event_log.go:115  [n1] Event: "alter_table", target: 4, info: {TableName:users Statement:ALTER TABLE system.users ADD COLUMN IF NOT EXISTS "isRole" BOOL NOT NULL DEFAULT false User:node MutationID:1 CascadeDroppedViews:[]}
I180116 16:28:48.214201 157820 sql/lease.go:348  [n1] publish: descID=4 (users) version=2 mtime=2018-01-16 16:28:48.213891792 +0000 UTC
I180116 16:28:48.215557 158031 storage/replica_command.go:819  [split,n1,s1,r14/1:/{Table/17-Max}] initiating a split of this range at key /Table/18 [r15]
I180116 16:28:48.218811 158031 storage/replica_command.go:819  [split,n1,s1,r15/1:/{Table/18-Max}] initiating a split of this range at key /Table/19 [r16]
I180116 16:28:48.223647 157820 sql/lease.go:348  [n1] publish: descID=4 (users) version=3 mtime=2018-01-16 16:28:48.223224643 +0000 UTC
I180116 16:28:48.227994 158031 storage/replica_command.go:819  [split,n1,s1,r16/1:/{Table/19-Max}] initiating a split of this range at key /Table/20 [r17]
I180116 16:28:48.228590 157820 sql/backfill.go:132  [n1] Running backfill for "users", v=3, m=1
I180116 16:28:48.231527 158031 storage/replica_command.go:819  [split,n1,s1,r17/1:/{Table/20-Max}] initiating a split of this range at key /Table/21 [r18]
I180116 16:28:48.234972 158031 storage/replica_command.go:819  [split,n1,s1,r18/1:/{Table/21-Max}] initiating a split of this range at key /Table/22 [r19]
W180116 16:28:48.237337 158299 storage/intent_resolver.go:324  [n1,s1] failed to push during intent resolution: failed to push "split" id=710285b3 key=/Local/Range/Table/21/RangeDescriptor rw=true pri=0.03966754 iso=SERIALIZABLE stat=PENDING epo=0 ts=1516120128.234995043,0 orig=1516120128.234995043,0 max=1516120128.234995043,0 wto=false rop=false seq=3
I180116 16:28:48.238261 157820 sql/lease.go:348  [n1] publish: descID=4 (users) version=4 mtime=2018-01-16 16:28:48.238167096 +0000 UTC
I180116 16:28:48.238595 158031 storage/replica_command.go:819  [split,n1,s1,r19/1:/{Table/22-Max}] initiating a split of this range at key /Table/23 [r20]
I180116 16:28:48.241703 157820 sql/event_log.go:115  [n1] Event: "finish_schema_change", target: 4, info: {MutationID:1}
I180116 16:28:48.242876 157820 sql/lease.go:274  publish (count leases): descID=4 name=users version=3 count=1
I180116 16:28:48.273211 157820 server/server.go:1232  [n1] done ensuring all necessary migrations have run
I180116 16:28:48.273253 157820 server/server.go:1235  [n1] serving sql connections
I180116 16:28:48.275247 158355 sql/event_log.go:115  [n1] Event: "node_join", target: 1, info: {Descriptor:{NodeID:1 Address:{NetworkField:tcp AddressField:127.0.0.1:44949} Attrs: Locality: ServerVersion:1.1-8} ClusterID:f8371979-8bbb-42c2-9861-24b11bebe89a StartedAt:1516120128149480129 LastUp:1516120128149480129}
W180116 16:28:48.279050 157820 server/status/runtime.go:109  Could not parse build timestamp: parsing time "" as "2006/01/02 15:04:05": cannot parse "" as "2006"
I180116 16:28:48.281448 157820 server/config.go:516  [n?] 1 storage engine initialized
I180116 16:28:48.281471 157820 server/config.go:519  [n?] RocksDB cache size: 128 MiB
I180116 16:28:48.281477 157820 server/config.go:519  [n?] store 0: in-memory, size 0 B
W180116 16:28:48.281540 157820 gossip/gossip.go:1292  [n?] no incoming or outgoing connections
I180116 16:28:48.281575 157820 server/server.go:936  [n?] no stores bootstrapped and --join flag specified, awaiting init command.
I180116 16:28:48.291341 158204 gossip/client.go:129  [n?] started gossip client to 127.0.0.1:44949
I180116 16:28:48.293688 158328 gossip/server.go:219  [n1] received initial cluster-verification connection from {tcp 127.0.0.1:45903}
I180116 16:28:48.297171 157820 storage/stores.go:331  [n?] read 0 node addresses from persistent storage
I180116 16:28:48.297224 157820 storage/stores.go:350  [n?] wrote 1 node addresses to persistent storage
I180116 16:28:48.297234 157820 server/node.go:627  [n?] connecting to gossip network to verify cluster ID...
I180116 16:28:48.297258 157820 server/node.go:652  [n?] node connected via gossip and verified as part of cluster "f8371979-8bbb-42c2-9861-24b11bebe89a"
I180116 16:28:48.297482 158351 kv/dist_sender.go:354  [n?] unable to determine this node's attributes for replica selection; node is most likely bootstrapping
I180116 16:28:48.298614 158350 kv/dist_sender.go:354  [n?] unable to determine this node's attributes for replica selection; node is most likely bootstrapping
I180116 16:28:48.299487 157820 kv/dist_sender.go:354  [n?] unable to determine this node's attributes for replica selection; node is most likely bootstrapping
I180116 16:28:48.300547 157820 server/node.go:332  [n?] new node allocated ID 2
I180116 16:28:48.300616 157820 gossip/gossip.go:332  [n2] NodeDescriptor set to node_id:2 address:<network_field:"tcp" address_field:"127.0.0.1:45903" > attrs:<> locality:<> ServerVersion:<major_val:1 minor_val:1 patch:0 unstable:8 > 
I180116 16:28:48.300675 157820 server/node.go:414  [n2] node=2: asynchronously bootstrapping engine(s) [<no-attributes>=<in-mem>]
I180116 16:28:48.300697 157820 server/node.go:428  [n2] node=2: started with [] engine(s) and attributes []
I180116 16:28:48.300806 157820 sql/distsql_physical_planner.go:122  [n2] creating DistSQLPlanner with address {tcp 127.0.0.1:45903}
I180116 16:28:48.301387 158376 storage/stores.go:350  [n1] wrote 1 node addresses to persistent storage
I180116 16:28:48.303309 157820 server/server.go:1161  [n2] starting https server at 127.0.0.1:45763
I180116 16:28:48.303336 157820 server/server.go:1162  [n2] starting grpc/postgres server at 127.0.0.1:45903
I180116 16:28:48.303345 157820 server/server.go:1163  [n2] advertising CockroachDB node at 127.0.0.1:45903
I180116 16:28:48.303574 158392 server/node.go:608  [n2] bootstrapped store [n2,s2]
I180116 16:28:48.304539 157820 server/server.go:1232  [n2] done ensuring all necessary migrations have run
I180116 16:28:48.304569 157820 server/server.go:1235  [n2] serving sql connections
I180116 16:28:48.309154 158336 sql/event_log.go:115  [n2] Event: "node_join", target: 2, info: {Descriptor:{NodeID:2 Address:{NetworkField:tcp AddressField:127.0.0.1:45903} Attrs: Locality: ServerVersion:1.1-8} ClusterID:f8371979-8bbb-42c2-9861-24b11bebe89a StartedAt:1516120128300687650 LastUp:1516120128300687650}
W180116 16:28:48.311086 157820 server/status/runtime.go:109  Could not parse build timestamp: parsing time "" as "2006/01/02 15:04:05": cannot parse "" as "2006"
I180116 16:28:48.315364 157820 server/config.go:516  [n?] 1 storage engine initialized
I180116 16:28:48.315389 157820 server/config.go:519  [n?] RocksDB cache size: 128 MiB
I180116 16:28:48.315396 157820 server/config.go:519  [n?] store 0: in-memory, size 0 B
W180116 16:28:48.315467 157820 gossip/gossip.go:1292  [n?] no incoming or outgoing connections
I180116 16:28:48.315526 157820 server/server.go:936  [n?] no stores bootstrapped and --join flag specified, awaiting init command.
I180116 16:28:48.326818 158411 gossip/client.go:129  [n?] started gossip client to 127.0.0.1:44949
I180116 16:28:48.327185 158596 gossip/server.go:219  [n1] received initial cluster-verification connection from {tcp 127.0.0.1:41455}
I180116 16:28:48.328204 157820 storage/stores.go:331  [n?] read 0 node addresses from persistent storage
I180116 16:28:48.328249 157820 storage/stores.go:350  [n?] wrote 2 node addresses to persistent storage
I180116 16:28:48.328261 157820 server/node.go:627  [n?] connecting to gossip network to verify cluster ID...
I180116 16:28:48.328284 157820 server/node.go:652  [n?] node connected via gossip and verified as part of cluster "f8371979-8bbb-42c2-9861-24b11bebe89a"
I180116 16:28:48.328524 158618 kv/dist_sender.go:354  [n?] unable to determine this node's attributes for replica selection; node is most likely bootstrapping
I180116 16:28:48.329371 158617 kv/dist_sender.go:354  [n?] unable to determine this node's attributes for replica selection; node is most likely bootstrapping
I180116 16:28:48.330119 157820 kv/dist_sender.go:354  [n?] unable to determine this node's attributes for replica selection; node is most likely bootstrapping
I180116 16:28:48.330943 157820 server/node.go:332  [n?] new node allocated ID 3
I180116 16:28:48.331011 157820 gossip/gossip.go:332  [n3] NodeDescriptor set to node_id:3 address:<network_field:"tcp" address_field:"127.0.0.1:41455" > attrs:<> locality:<> ServerVersion:<major_val:1 minor_val:1 patch:0 unstable:8 > 
I180116 16:28:48.331112 157820 server/node.go:414  [n3] node=3: asynchronously bootstrapping engine(s) [<no-attributes>=<in-mem>]
I180116 16:28:48.331133 157820 server/node.go:428  [n3] node=3: started with [] engine(s) and attributes []
I180116 16:28:48.331187 157820 sql/distsql_physical_planner.go:122  [n3] creating DistSQLPlanner with address {tcp 127.0.0.1:41455}
I180116 16:28:48.332045 158569 storage/stores.go:350  [n1] wrote 2 node addresses to persistent storage
I180116 16:28:48.332494 158378 storage/stores.go:350  [n2] wrote 2 node addresses to persistent storage
I180116 16:28:48.333695 158621 server/node.go:608  [n3] bootstrapped store [n3,s3]
I180116 16:28:48.334336 157820 server/server.go:1161  [n3] starting https server at 127.0.0.1:38685
I180116 16:28:48.334361 157820 server/server.go:1162  [n3] starting grpc/postgres server at 127.0.0.1:41455
I180116 16:28:48.334371 157820 server/server.go:1163  [n3] advertising CockroachDB node at 127.0.0.1:41455
I180116 16:28:48.336042 157820 server/server.go:1232  [n3] done ensuring all necessary migrations have run
I180116 16:28:48.336068 157820 server/server.go:1235  [n3] serving sql connections
I180116 16:28:48.336926 157837 storage/replica_raftstorage.go:540  [replicate,n1,s1,r14/1:/Table/1{7-8}] generated preemptive snapshot f64aec7c at index 16
I180116 16:28:48.338143 157820 testutils/testcluster/testcluster.go:528  [n1,s1] has 20 underreplicated ranges
I180116 16:28:48.341425 158772 sql/event_log.go:115  [n3] Event: "node_join", target: 3, info: {Descriptor:{NodeID:3 Address:{NetworkField:tcp AddressField:127.0.0.1:41455} Attrs: Locality: ServerVersion:1.1-8} ClusterID:f8371979-8bbb-42c2-9861-24b11bebe89a StartedAt:1516120128331124840 LastUp:1516120128331124840}
I180116 16:28:48.347632 157837 storage/store.go:3567  [replicate,n1,s1,r14/1:/Table/1{7-8}] streamed snapshot to (n2,s2):?: kv pairs: 10, log entries: 6, rate-limit: 8.0 MiB/sec, 1ms
I180116 16:28:48.347635 157820 testutils/testcluster/testcluster.go:528  [n1,s1] has 20 underreplicated ranges
I180116 16:28:48.347987 158803 storage/replica_raftstorage.go:746  [n2,s2,r14/?:{-}] applying preemptive snapshot at index 16 (id=f64aec7c, encoded size=2476, 1 rocksdb batches, 6 log entries)
I180116 16:28:48.348263 158803 storage/replica_raftstorage.go:752  [n2,s2,r14/?:/Table/1{7-8}] applied preemptive snapshot in 0ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I180116 16:28:48.348887 157837 storage/replica_command.go:1741  [replicate,n1,s1,r14/1:/Table/1{7-8}] change replicas (ADD_REPLICA (n2,s2):2): read existing descriptor r14:/Table/1{7-8} [(n1,s1):1, next=2]
I180116 16:28:48.350793 157837 storage/replica.go:3185  [n1,s1,r14/1:/Table/1{7-8}] proposing ADD_REPLICA((n2,s2):2): updated=[(n1,s1):1 (n2,s2):2] next=3
I180116 16:28:48.351402 157837 storage/replica_raftstorage.go:540  [replicate,n1,s1,r1/1:/{Min-System/}] generated preemptive snapshot e1f75307 at index 71
I180116 16:28:48.352589 158823 storage/raft_transport.go:459  [n2] raft transport stream to node 1 established
I180116 16:28:48.361942 157837 storage/store.go:3567  [replicate,n1,s1,r1/1:/{Min-System/}] streamed snapshot to (n3,s3):?: kv pairs: 54, log entries: 48, rate-limit: 8.0 MiB/sec, 1ms
I180116 16:28:48.362387 158559 storage/replica_raftstorage.go:746  [n3,s3,r1/?:{-}] applying preemptive snapshot at index 71 (id=e1f75307, encoded size=13369, 1 rocksdb batches, 48 log entries)
I180116 16:28:48.363289 158559 storage/replica_raftstorage.go:752  [n3,s3,r1/?:/{Min-System/}] applied preemptive snapshot in 1ms [clear=0ms batch=0ms entries=0ms commit=1ms]
I180116 16:28:48.363942 157837 storage/replica_command.go:1741  [replicate,n1,s1,r1/1:/{Min-System/}] change replicas (ADD_REPLICA (n3,s3):2): read existing descriptor r1:/{Min-System/} [(n1,s1):1, next=2]
I180116 16:28:48.366319 157837 storage/replica.go:3185  [n1,s1,r1/1:/{Min-System/}] proposing ADD_REPLICA((n3,s3):2): updated=[(n1,s1):1 (n3,s3):2] next=3
I180116 16:28:48.366959 158032 storage/replica_raftstorage.go:540  [replicate,n1,s1,r1/1:/{Min-System/}] generated preemptive snapshot 00ee5272 at index 74
I180116 16:28:48.367888 158032 storage/store.go:3567  [replicate,n1,s1,r1/1:/{Min-System/}] streamed snapshot to (n2,s2):?: kv pairs: 57, log entries: 51, rate-limit: 8.0 MiB/sec, 1ms
I180116 16:28:48.367926 158765 storage/raft_transport.go:459  [n3] raft transport stream to node 1 established
I180116 16:28:48.368247 158760 storage/replica_raftstorage.go:746  [n2,s2,r1/?:{-}] applying preemptive snapshot at index 74 (id=00ee5272, encoded size=14974, 1 rocksdb batches, 51 log entries)
I180116 16:28:48.368561 158760 storage/replica_raftstorage.go:752  [n2,s2,r1/?:/{Min-System/}] applied preemptive snapshot in 0ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I180116 16:28:48.369087 158032 storage/replica_command.go:1741  [replicate,n1,s1,r1/1:/{Min-System/}] change replicas (ADD_REPLICA (n2,s2):3): read existing descriptor r1:/{Min-System/} [(n1,s1):1, (n3,s3):2, next=3]
I180116 16:28:48.369837 157820 testutils/testcluster/testcluster.go:528  [n1,s1] has 20 underreplicated ranges
I180116 16:28:48.371442 158776 storage/replica.go:3185  [n1,s1,r1/1:/{Min-System/}] proposing ADD_REPLICA((n2,s2):3): updated=[(n1,s1):1 (n3,s3):2 (n2,s2):3] next=4
I180116 16:28:48.372556 157837 storage/replica_raftstorage.go:540  [replicate,n1,s1,r6/1:/{System/tse-Table/System…}] generated preemptive snapshot 3f2e384e at index 19
panic: test timed out after 4m0s

goroutine 158843 [running]:
testing.startAlarm.func1()
	/usr/local/go/src/testing/testing.go:1145 +0xf9
created by time.goFunc
	/usr/local/go/src/time/sleep.go:170 +0x44

goroutine 1 [chan receive]:
testing.(*T).Run(0xc4200a0780, 0x230e293, 0x1e, 0x23c7f38, 0x8d4f01)
	/usr/local/go/src/testing/testing.go:790 +0x2fc
testing.runTests.func1(0xc4200a0780)
	/usr/local/go/src/testing/testing.go:1004 +0x64
testing.tRunner(0xc4200a0780, 0xc4206a5db8)
	/usr/local/go/src/testing/testing.go:746 +0xd0
testing.runTests(0xc4202a2080, 0x33b3600, 0x38, 0x38, 0x0)
	/usr/local/go/src/testing/testing.go:1002 +0x2d8
testing.(*M).Run(0xc42092bf18, 0x23c81c8)
	/usr/local/go/src/testing/testing.go:921 +0x111
github.com/cockroachdb/cockroach/pkg/ccl/sqlccl.TestMain(0xc4206a5f18)
	/go/src/github.com/cockroachdb/cockroach/pkg/ccl/sqlccl/main_test.go:31 +0xda
main.main()
	github.com/cockroachdb/cockroach/pkg/ccl/sqlccl/_test/_testmain.go:172 +0xdb

goroutine 6 [chan receive]:
github.com/cockroachdb/cockroach/pkg/util/log.(*loggingT).flushDaemon(0x3600940)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:1043 +0x81
created by github.com/cockroachdb/cockroach/pkg/util/log.init.0
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:581 +0xbf

goroutine 23 [syscall, 4 minutes]:
os/signal.signal_recv(0x0)
	/usr/local/go/src/runtime/sigqueue.go:131 +0xa6
os/signal.loop()
	/usr/local/go/src/os/signal/signal_unix.go:22 +0x22
created by os/signal.init.0
	/usr/local/go/src/os/signal/signal_unix.go:28 +0x41

goroutine 148702 [chan receive]:
testing.(*T).Run(0xc422d000f0, 0x22e6b00, 0x7, 0xc42041aec0, 0x1)
	/usr/local/go/src/testing/testing.go:790 +0x2fc
github.com/cockroachdb/cockroach/pkg/ccl/sqlccl_test.TestRestoreDatabaseVersusTable(0xc422d000f0)
	/go/src/github.com/cockroachdb/cockroach/pkg/ccl/sqlccl/backup_test.go:2217 +0x808
testing.tRunner(0xc422d000f0, 0x23c7f38)
	/usr/local/go/src/testing/testing.go:746 +0xd0
created by testing.(*T).Run
	/usr/local/go/src/testing/testing.go:789 +0x2de

goroutine 52 [select, locked to thread]:
runtime.gopark(0x23cbcc0, 0x0, 0x22e54ff, 0x6, 0x18, 0x1)
	/usr/local/go/src/runtime/proc.go:277 +0x12c
runtime.selectgo(0xc42003ff50, 0xc42044ac00)
	/usr/local/go/src/runtime/select.go:395 +0x1138
runtime.ensureSigM.func1()
	/usr/local/go/src/runtime/signal_unix.go:511 +0x220
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:2337 +0x1

goroutine 158708 [semacquire]:
sync.runtime_notifyListWait(0xc422edce50, 0xc40000003b)
	/usr/local/go/src/runtime/sema.go:507 +0x110
sync.(*Cond).Wait(0xc422edce40)
	/usr/local/go/src/sync/cond.go:56 +0x80
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc4224a1290, 0x33de160, 0xc421c0cd50)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:197 +0x7c
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x33de160, 0xc421c0cd50)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:166 +0x3e
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4202db080, 0xc4225a1950, 0xc4202db070)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0xf3
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:188 +0xad

goroutine 158505 [semacquire]:
sync.runtime_notifyListWait(0xc422422350, 0xc400000036)
	/usr/local/go/src/runtime/sema.go:507 +0x110
sync.(*Cond).Wait(0xc422422340)
	/usr/local/go/src/sync/cond.go:56 +0x80
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc421996240, 0x33de160, 0xc4226530b0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:197 +0x7c
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x33de160, 0xc4226530b0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:166 +0x3e
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4208ec9a0, 0xc421bc1050, 0xc4208ec990)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0xf3
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:188 +0xad

goroutine 158333 [select]:
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.(*http2Server).keepalive(0xc421dfe160)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/http2_server.go:935 +0x264
created by github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.newHTTP2Server
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/http2_server.go:230 +0x8fa

goroutine 158015 [semacquire]:
sync.runtime_notifyListWait(0xc422242450, 0xc40000016b)
	/usr/local/go/src/runtime/sema.go:507 +0x110
sync.(*Cond).Wait(0xc422242440)
	/usr/local/go/src/sync/cond.go:56 +0x80
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc4226081b0, 0x33de160, 0xc4213c6a50)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:197 +0x7c
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x33de160, 0xc4213c6a50)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:166 +0x3e
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4203c6c90, 0xc4228a1a70, 0xc4203c6c80)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0xf3
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:188 +0xad

goroutine 158070 [select]:
github.com/cockroachdb/cockroach/pkg/sql.(*SchemaChangeManager).Start.func1(0x33de160, 0xc422d02a20)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/schema_changer.go:1023 +0x9b6
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4214b6660, 0xc4228a1a70, 0xc4202be240)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0xf3
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:188 +0xad

goroutine 158396 [select]:
github.com/cockroachdb/cockroach/pkg/server.(*Server).startSampleEnvironment.func1(0x33de160, 0xc420846ff0)
	/go/src/github.com/cockroachdb/cockroach/pkg/server/server.go:1416 +0x16c
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc420e0a540, 0xc421bc1050, 0xc423a7ef80)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0xf3
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:188 +0xad

goroutine 158468 [semacquire]:
sync.runtime_notifyListWait(0xc422422350, 0xc400000033)
	/usr/local/go/src/runtime/sema.go:507 +0x110
sync.(*Cond).Wait(0xc422422340)
	/usr/local/go/src/sync/cond.go:56 +0x80
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc421996240, 0x33de160, 0xc423b6c7b0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:197 +0x7c
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x33de160, 0xc423b6c7b0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:166 +0x3e
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4208ec500, 0xc421bc1050, 0xc4208ec4f0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0xf3
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:188 +0xad

goroutine 149011 [semacquire]:
sync.runtime_notifyListWait(0xc421b20590, 0xc400000236)
	/usr/local/go/src/runtime/sema.go:507 +0x110
sync.(*Cond).Wait(0xc421b20580)
	/usr/local/go/src/sync/cond.go:56 +0x80
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc4221ce240, 0x33de160, 0xc4225f87e0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:197 +0x7c
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x33de160, 0xc4225f87e0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:166 +0x3e
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc420400560, 0xc422db45a0, 0xc420400550)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0xf3
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:188 +0xad

goroutine 148982 [semacquire]:
sync.runtime_notifyListWait(0xc421b20590, 0xc400000249)
	/usr/local/go/src/runtime/sema.go:507 +0x110
sync.(*Cond).Wait(0xc421b20580)
	/usr/local/go/src/sync/cond.go:56 +0x80
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc4221ce240, 0x33de160, 0xc42446a120)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:197 +0x7c
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x33de160, 0xc42446a120)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:166 +0x3e
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc420400090, 0xc422db45a0, 0xc420400080)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0xf3
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:188 +0xad

goroutine 158005 [semacquire]:
sync.runtime_notifyListWait(0xc422242450, 0xc400000170)
	/usr/local/go/src/runtime/sema.go:507 +0x110
sync.(*Cond).Wait(0xc422242440)
	/usr/local/go/src/sync/cond.go:56 +0x80
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc4226081b0, 0x33de160, 0xc4229586f0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:197 +0x7c
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x33de160, 0xc4229586f0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:166 +0x3e
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4203c6ab0, 0xc4228a1a70, 0xc4203c6aa0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0xf3
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:188 +0xad

goroutine 158624 [select]:
github.com/cockroachdb/cockroach/pkg/server.(*Server).refreshSettings.func2(0x33de160, 0xc42518a510)
	/go/src/github.com/cockroachdb/cockroach/pkg/server/settingsworker.go:112 +0x2db
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc423ab7b60, 0xc4225a1950, 0xc4203507e0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0xf3
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:188 +0xad

goroutine 158440 [chan receive]:
github.com/cockroachdb/cockroach/vendor/github.com/cockroachdb/cmux.muxListener.Accept(...)
	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/cockroachdb/cmux/cmux.go:184
github.com/cockroachdb/cockroach/vendor/github.com/cockroachdb/cmux.(*muxListener).Accept(0xc421a56540, 0xc425609530, 0xc42475be80, 0x96d4e5, 0xc425609558)
	<autogenerated>:1 +0x65
crypto/tls.(*listener).Accept(0xc421a565a0, 0xc420014108, 0x2068180, 0x3397560, 0x22ae720)
	/usr/local/go/src/crypto/tls/tls.go:52 +0x37
net/http.(*Server).Serve(0xc42139be10, 0x33cf8a0, 0xc421a565a0, 0x0, 0x0)
	/usr/local/go/src/net/http/server.go:2695 +0x1b2
github.com/cockroachdb/cockroach/pkg/server.(*Server).Start.func5(0x33de160, 0xc4256094d0)
	/go/src/github.com/cockroachdb/cockroach/pkg/server/server.go:779 +0x42
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4203c6880, 0xc4225a1950, 0xc421a565c0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0xf3
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:188 +0xad

goroutine 158623 [select]:
github.com/cockroachdb/cockroach/pkg/server.(*Node).startGossip.func1(0x33de160, 0xc42518a4e0)
	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:680 +0x2dd
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc423ab7b30, 0xc4225a1950, 0xc4203507c0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0xf3
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:188 +0xad

goroutine 158370 [chan receive]:
github.com/cockroachdb/cockroach/vendor/github.com/cockroachdb/cmux.muxListener.Accept(...)
	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/cockroachdb/cmux/cmux.go:184
github.com/cockroachdb/cockroach/vendor/github.com/cockroachdb/cmux.(*muxListener).Accept(0xc422adae20, 0x23cb0a0, 0xc4209328c0, 0x33e9f60, 0xc42324eaa0)
	<autogenerated>:1 +0x65
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.(*Server).Serve(0xc4209328c0, 0x33dd2e0, 0xc422adae20, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:463 +0x196
github.com/cockroachdb/cockroach/pkg/server.(*Server).Start.func7(0x33de160, 0xc4214a0fc0)
	/go/src/github.com/cockroachdb/cockroach/pkg/server/server.go:795 +0x43
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc420c96c20, 0xc421bc1050, 0xc422adaf40)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0xf3
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:188 +0xad

goroutine 158496 [semacquire]:
sync.runtime_notifyListWait(0xc422422350, 0xc400000024)
	/usr/local/go/src/runtime/sema.go:507 +0x110
sync.(*Cond).Wait(0xc422422340)
	/usr/local/go/src/sync/cond.go:56 +0x80
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc421996240, 0x33de160, 0xc422652ed0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:197 +0x7c
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x33de160, 0xc422652ed0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:166 +0x3e
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4208ec880, 0xc421bc1050, 0xc4208ec870)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0xf3
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:188 +0xad

goroutine 158271 [semacquire]:
sync.runtime_notifyListWait(0xc422422350, 0xc400000006)
	/usr/local/go/src/runtime/sema.go:507 +0x110
sync.(*Cond).Wait(0xc422422340)
	/usr/local/go/src/sync/cond.go:56 +0x80
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc421996240, 0x33de160, 0xc423b6c360)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:197 +0x7c
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x33de160, 0xc423b6c360)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:166 +0x3e
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4208ec260, 0xc421bc1050, 0xc4208ec250)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0xf3
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:188 +0xad

goroutine 158098 [select]:
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.(*http2Client).keepalive(0xc420c48d80)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/http2_client.go:1266 +0x14b
created by github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.newHTTP2Client
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/http2_client.go:302 +0xe51

goroutine 158796 [select]:
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.(*recvBufferReader).read(0xc4213d3270, 0xc4231bafd0, 0x5, 0x5, 0xf, 0xc4238fbb48, 0x8719e8)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/transport.go:133 +0x28b
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.(*recvBufferReader).Read(0xc4213d3270, 0xc4231bafd0, 0x5, 0x5, 0x3, 0xc420022000, 0xc420022070)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/transport.go:122 +0x67
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.(*transportReader).Read(0xc422026f30, 0xc4231bafd0, 0x5, 0x5, 0x8bcfce, 0xc421e010c8, 0xc421af6100)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/transport.go:403 +0x55
io.ReadAtLeast(0x33c43a0, 0xc422026f30, 0xc4231bafd0, 0x5, 0x5, 0x5, 0xc4221e6580, 0xc421e01000, 0xc400000005)
	/usr/local/go/src/io/io.go:309 +0x86
io.ReadFull(0x33c43a0, 0xc422026f30, 0xc4231bafd0, 0x5, 0x5, 0x87495b, 0xc4238fbc40, 0x89ff20)
	/usr/local/go/src/io/io.go:327 +0x58
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.(*Stream).Read(0xc421e01000, 0xc4231bafd0, 0x5, 0x5, 0xc4238fbcc0, 0x88770a, 0xc4222663c0)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/transport.go:387 +0xbf
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.(*parser).recvMsg(0xc4231bafc0, 0x7fffffff, 0x60, 0x58, 0x220a4a0, 0xc4238fbdb8, 0xe0000000006, 0x58)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/rpc_util.go:270 +0x65
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.recv(0xc4231bafc0, 0x33dd960, 0x364bc88, 0xc421e01000, 0x33cdf60, 0x364bc88, 0x21c71a0, 0xc42040a4a0, 0x7fffffff, 0xc42378c7e0, ...)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/rpc_util.go:356 +0x4d
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.(*serverStream).RecvMsg(0xc4237e2680, 0x21c71a0, 0xc42040a4a0, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/stream.go:644 +0x10e
github.com/cockroachdb/cockroach/pkg/storage.(*multiRaftRaftMessageBatchServer).Recv(0xc4208ecdc0, 0x33de160, 0xc422026fc0, 0xc424d00900)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/raft.pb.go:394 +0x62
github.com/cockroachdb/cockroach/pkg/storage.(*RaftTransport).RaftMessageBatch.func1.1.1(0x33e9e40, 0xc4208ecdc0, 0xc4221dc5a0, 0x33de160, 0xc422026fc0, 0x18, 0xfe67b0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/raft_transport.go:330 +0x83
github.com/cockroachdb/cockroach/pkg/storage.(*RaftTransport).RaftMessageBatch.func1.1(0x33de160, 0xc422026fc0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/raft_transport.go:353 +0x5d
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4208ecde0, 0xc4228a1a70, 0xc422026f90)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0xf3
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:188 +0xad

goroutine 158720 [select]:
github.com/cockroachdb/cockroach/pkg/storage.(*Store).startGossip.func4(0x33de160, 0xc421c0d020)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/store.go:1313 +0x3e9
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4202db1d0, 0xc4225a1950, 0xc420f92640)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0xf3
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:188 +0xad

goroutine 158821 [select]:
github.com/cockroachdb/cockroach/pkg/storage.(*RaftTransport).RaftMessageBatch(0xc422d01950, 0x33e9e40, 0xc420e0be60, 0x33cb720, 0xc422d01950)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/raft_transport.go:359 +0x215
github.com/cockroachdb/cockroach/pkg/storage._MultiRaft_RaftMessageBatch_Handler(0x22a9c60, 0xc422d01950, 0x33e7200, 0xc420a36980, 0x8580c8, 0x20)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/raft.pb.go:375 +0xb2
github.com/cockroachdb/cockroach/pkg/rpc.NewServerWithInterceptor.func2(0x22a9c60, 0xc422d01950, 0x33e7200, 0xc420a36980, 0xc421a57b80, 0x23c9b20, 0x40, 0x3f)
	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:187 +0x103
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.(*Server).processStreamingRPC(0xc4209328c0, 0x33e92a0, 0xc4238e0160, 0xc423370900, 0xc422822390, 0x33a50c0, 0x0, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:940 +0x2e8
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.(*Server).handleStream(0xc4209328c0, 0x33e92a0, 0xc4238e0160, 0xc423370900, 0x0)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:1027 +0x14c1
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.(*Server).serveStreams.func1.1(0xc4202cc500, 0xc4209328c0, 0x33e92a0, 0xc4238e0160, 0xc423370900)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:572 +0x9f
created by github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.(*Server).serveStreams.func1
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:570 +0xa1

goroutine 158364 [chan receive]:
github.com/cockroachdb/cockroach/pkg/util/netutil.MakeServer.func2(0x33de160, 0xc4208df9b0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/netutil/net.go:100 +0x64
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc420c96b40, 0xc421bc1050, 0xc422adad40)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0xf3
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:188 +0xad

goroutine 158668 [semacquire]:
sync.runtime_notifyListWait(0xc422edce50, 0xc400000013)
	/usr/local/go/src/runtime/sema.go:507 +0x110
sync.(*Cond).Wait(0xc422edce40)
	/usr/local/go/src/sync/cond.go:56 +0x80
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc4224a1290, 0x33de160, 0xc421c0c450)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:197 +0x7c
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x33de160, 0xc421c0c450)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:166 +0x3e
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4202da9f0, 0xc4225a1950, 0xc4202da9d0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0xf3
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:188 +0xad

goroutine 157984 [semacquire]:
sync.runtime_notifyListWait(0xc422242450, 0xc400000169)
	/usr/local/go/src/runtime/sema.go:507 +0x110
sync.(*Cond).Wait(0xc422242440)
	/usr/local/go/src/sync/cond.go:56 +0x80
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc4226081b0, 0x33de160, 0xc4213c6990)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:197 +0x7c
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x33de160, 0xc4213c6990)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:166 +0x3e
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4203c6780, 0xc4228a1a70, 0xc4203c6770)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0xf3
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:188 +0xad

goroutine 158319 [select]:
github.com/cockroachdb/cockroach/pkg/storage.(*baseQueue).processLoop.func1(0x33de160, 0xc421c0d140)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/queue.go:514 +0x1bb
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4208ec080, 0xc4225a1950, 0xc4203de180)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0xf3
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:188 +0xad

goroutine 158019 [semacquire]:
sync.runtime_notifyListWait(0xc422242450, 0xc40000017c)
	/usr/local/go/src/runtime/sema.go:507 +0x110
sync.(*Cond).Wait(0xc422242440)
	/usr/local/go/src/sync/cond.go:56 +0x80
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc4226081b0, 0x33de160, 0xc422d02690)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:197 +0x7c
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x33de160, 0xc422d02690)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:166 +0x3e
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4203c6d10, 0xc4228a1a70, 0xc4203c6d00)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:195 +0xf3
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:188 +0xad

goroutine 158596 [select]:
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.(*recvBufferReader).read(0xc421a615e0, 0xc422adb050, 0x5, 0x5, 0xc42042f301, 0xc421dfd830, 0x8719e8)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/transport.go:133 +0x28b
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.(*recvBufferReader).Read(0xc421a615e0, 0xc422adb050, 0x5, 0x5, 0x3, 0xc420024600, 0xc420024670)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/transport.go:122 +0x67
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.(*transportReader).Read(0xc4245636e0, 0xc422adb050, 0x5, 0x5, 0x8bcfce, 0xc4201ce4c8, 0xc422a99580)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.gola
```
Please assign, take a look and update the issue accordingly.

--- a/pkg/ccl/sqlccl/partition_test.go
+++ b/pkg/ccl/sqlccl/partition_test.go
@@ -1061,18 +1061,14 @@ func verifyScansOnNode(db *gosql.DB, query string, node string) error {
 	return nil
 }
 
-func TestInitialPartitioning(t *testing.T) {
-	defer leaktest.AfterTest(t)()
-	rng, _ := randutil.NewPseudoRand()
-
+func setupPartitioningTestCluster(ctx context.Context, t testing.TB) (*sqlutils.SQLRunner, func()) {
 	cfg := config.DefaultZoneConfig()
 	cfg.NumReplicas = 1
-	defer config.TestingSetDefaultZoneConfig(cfg)()
+	resetZoneConfig := config.TestingSetDefaultZoneConfig(cfg)
 
-	ctx := context.Background()
 	tsArgs := func(attr string) base.TestServerArgs {
 		return base.TestServerArgs{
-			ScanInterval: time.Second,
+			ScanInterval: 100 * time.Millisecond,
 			StoreSpecs: []base.StoreSpec{
 				{InMemory: true, Attributes: roachpb.Attributes{Attrs: []string{attr}}},
 			},
@@ -1084,12 +1080,30 @@ func TestInitialPartitioning(t *testing.T) {
 		2: tsArgs("n3"),
 	}}
 	tc := testcluster.StartTestCluster(t, 3, tcArgs)
-	defer tc.Stopper().Stop(context.Background())
+
 	sqlDB := sqlutils.MakeSQLRunner(tc.Conns[0])
 	sqlDB.Exec(t, `CREATE DATABASE data`)
 	sqlDB.Exec(t, `USE data`)
 
+	// Disabling store throttling vastly speeds up rebalancing.
+	sqlDB.Exec(t, `SET CLUSTER SETTING server.declined_reservation_timeout = '0s'`)
+	sqlDB.Exec(t, `SET CLUSTER SETTING server.failed_reservation_timeout = '0s'`)
+
+	return sqlDB, func() {
+		tc.Stopper().Stop(context.Background())
+		resetZoneConfig()
+	}
+}
+
+func TestInitialPartitioning(t *testing.T) {
+	defer leaktest.AfterTest(t)()
+	rng, _ := randutil.NewPseudoRand()
 	testCases := allPartitioningTests(rng)
+
+	ctx := context.Background()
+	sqlDB, cleanup := setupPartitioningTestCluster(ctx, t)
+	defer cleanup()
+
 	for _, test := range testCases {
 		if len(test.scans) == 0 {
 			continue
@@ -1189,27 +1203,9 @@ func TestRepartitioning(t *testing.T) {
 		t.Fatalf("%+v", err)
 	}
 
-	cfg := config.DefaultZoneConfig()
-	cfg.NumReplicas = 1
-	defer config.TestingSetDefaultZoneConfig(cfg)()
-
 	ctx := context.Background()
-	tsArgs := func(attr string) base.TestServerArgs {
-		return base.TestServerArgs{
-			ScanInterval: time.Second,
-			StoreSpecs: []base.StoreSpec{
-				{InMemory: true, Attributes: roachpb.Attributes{Attrs: []string{attr}}},
-			},
-		}
-	}
-	tcArgs := base.TestClusterArgs{ServerArgsPerNode: map[int]base.TestServerArgs{
-		0: tsArgs("n1"),
-		1: tsArgs("n2"),
-		2: tsArgs("n3"),
-	}}
-	tc := testcluster.StartTestCluster(t, 3, tcArgs)
-	defer tc.Stopper().Stop(context.Background())
-	sqlDB := sqlutils.MakeSQLRunner(tc.Conns[0])
+	sqlDB, cleanup := setupPartitioningTestCluster(ctx, t)
+	defer cleanup()
 
 	for _, test := range testCases {
 		t.Run(fmt.Sprintf("%s/%s", test.old.name, test.new.name), func(t *testing.T) {

